src/components/timeline/FullTimeline.js

```diff
@@ -0,0 +1,332 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This file is the old timeline/index.js. It's not touched except the `Timeline` class rename